[SASS-11423] Add CYA page for Qualifying Loan Interest

MJCallahanPage opened the pull request:
**Description**
Adds the Check Answers Qualifying Loan Interest page

**JT**
- https://github.com/hmrc/income-tax-submission-journey-tests/pull/558

**PT**
- https://github.com/hmrc/income-tax-submission-performance-tests/pull/160

MJCallahanPage commented:
https://github.com/user-attachments/assets/6cfe653d-3de6-4357-8e15-9f183173ded1

baha-aboukoura commented on test/views/pages/businessTaxReliefs/CheckAnswersQualifyingLoanPageViewSpec.scala:
```diff
@@ -0,0 +1,80 @@
+
+    s"language is ${welshTest(userScenario.isWelsh)} and request is from an ${agentTest(userScenario.isAgent)}" should {
+
+      "render post-cessation trade relief amount page" which {
```
you have post cessation instead of qualifying loan interest

baha-aboukoura replied:
the name of this test is wrong

baha-aboukoura commented on app/views/pages/businessTaxReliefs/CheckAnswersQualifyingLoanPageView.scala.html:
```diff
@@ -0,0 +1,42 @@
+
+@import models.requests.JourneyDataRequest
+@import views.html.templates.Layout
+@import views.html.templates.helpers.{Button, ErrorSummary, Heading, InputText, Link}
```
unused imports

MJCallahanPage replied:
optimised

baha-aboukoura commented:
for some reason the [BusinessReliefsQualifyingLoanMessages](https://github.com/hmrc/income-tax-additional-info-frontend/blob/main/test/fixtures/messages/BusinessReliefsQualifyingLoanMessages.scala) test is sitting outside the businessTaxReliefs package

MJCallahanPage commented:
> for some reason the [BusinessReliefsQualifyingLoanMessages](https://github.com/hmrc/income-tax-additional-info-frontend/blob/main/test/fixtures/messages/BusinessReliefsQualifyingLoanMessages.scala) test is sitting outside the businessTaxReliefs package

Refactored and moved to package 👍

MJCallahanPage commented:
PR comments addressed on [9dc801c](https://github.com/hmrc/income-tax-additional-info-frontend/pull/120/commits/9dc801cf11e9c1aafb5ddd7a45a10177c8143faa)
